tests/Doctrine/Tests/DBAL/Schema/SqliteSchemaManagerTest.php

```diff
@@ -41,6 +40,61 @@ public function getDataColumnCollation()
             array(
                 'RTRIM', 'a"b', 'CREATE TABLE "a" ("a""b" text COLLATE RTRIM)'
             ),
+            array(
+                'BINARY', 'b', 'CREATE TABLE "a" (bb TEXT COLLATE RTRIM, b VARCHAR(42) NOT NULL COLLATE BINARY)'
+            ),
+            array(
+                'BINARY', 'b', 'CREATE TABLE "a" (bbb TEXT COLLATE NOCASE, bb TEXT COLLATE RTRIM, b VARCHAR(42) NOT NULL COLLATE BINARY)'
+            ),
+            array(
+                'BINARY', 'b', 'CREATE TABLE "a" (b VARCHAR(42) NOT NULL COLLATE BINARY, bb TEXT COLLATE RTRIM)'
+            ),
+        );
+    }
+
+    /**
+     * @dataProvider getDataColumnComment
+     */
+    public function testParseColumnCommentFromSQL($comment, $column, $sql)
+    {
+        $conn = $this->getMockBuilder('Doctrine\DBAL\Connection')->disableOriginalConstructor()->getMock();
+        $conn->expects($this->any())->method('getDatabasePlatform')->will($this->returnValue(new SqlitePlatform()));
```

Comment:
This can be simplified to `$conn->method('getDatabasePlatform')->willReturn(new SqlitePlatform());`